### DISCUSSION TITLE
Force update of field

### DISCUSSION THREAD

**Discussion by ewald1976** [filamentphp/filament#1304] [Help] [answered]
Hello,

i´d like to ask if there is a way to force an update of a field even if the value in the database is the same than in the edit form?
And I´d also would like to know if there is a way to access record data in the afterSave() hook?

Thank you in advance.

Kind regards.

**Comment by danharrin** [comment, marked as answer]
How would you force an update in normal Eloquent?

You can use `$this->record` to access the record data in `afterSave()` - or `$this->form->getState()` for the form data.

**Comment by ewald1976** [comment]
Understood. Thank you.
